docs/graphs/edge_attributes.rst

```diff
@@ -44,3 +44,77 @@ latitude and longitude coordinates of the source and target nodes.
        attributes:
          edge_length:
             _target_: anemoi.graphs.edges.attributes.EdgeDirection
+
+*********************
```

Comment:
I would group under the same header, so we get the left sidebar with the 3 types of attributes: Edge direction, edge length, attribute from node. Now:
![Screenshot 2024-12-19 at 09 08 05](https://github.com/user-attachments/assets/23dcf78e-44c6-4f7c-93b3-e7cb33009ad1)
I would drop the 1st and 3rd yaml examples, and also the subheaders. If you think it needs some explanation, you can also add a comment after a "#" in the yaml example.

Reply:
Done ✅